dfx.json

```diff
@@ -6,6 +6,11 @@
       "type": "custom",
       "wasm": "artifacts/waterneuron.wasm.gz"
```

Comment:
I looked into the artifacts folder and it is empty. Is it wise to let this path in the dfx.json?